src/System.Text.Encodings.Web/src/System/Text/Unicode/UnicodeRanges.cs

```diff
@@ -17,31 +17,31 @@ public static partial class UnicodeRanges
         /// An empty <see cref="UnicodeRange"/>. This range contains no code points.
         /// </summary>
         public static UnicodeRange None => _none ?? CreateEmptyRange(ref _none);
-        private static UnicodeRange _none;
+        private static UnicodeRange? _none;
 
         /// <summary>
         /// A <see cref="UnicodeRange"/> which contains all characters in the Unicode Basic
         /// Multilingual Plane (U+0000..U+FFFF).
         /// </summary>
         public static UnicodeRange All => _all ?? CreateRange(ref _all, '\u0000', '\uFFFF');
-        private static UnicodeRange _all;
+        private static UnicodeRange? _all;
 
         [MethodImpl(MethodImplOptions.NoInlining)] // the caller should be inlined, not this method
-        private static UnicodeRange CreateEmptyRange(ref UnicodeRange range)
+        private static UnicodeRange CreateEmptyRange(ref UnicodeRange? range)
```

Comment:
It may not matter given the usage, but this `range` parameter could be `[NotNull]`.